test/distributed/rpc/test_faulty_agent.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+import unittest
+
+from torch.testing._internal.common_distributed import MultiProcessTestCase
```

Comment:
Similar to my comment in https://github.com/pytorch/pytorch/pull/40816#discussion_r448060906, is it possible to avoid having separate files and enable different agents based on env variables?

Reply:
See my comment there.